src/lua_consolelib.c

```diff
@@ -385,13 +396,17 @@ static int lib_cvRegisterVar(lua_State *L)
 					if (lua_type(L, 5) != LUA_TSTRING
 					|| lua_type(L, 6) != LUA_TNUMBER)
 						FIELDERROR("PossibleValue", "custom PossibleValue table requires a format of string=integer, i.e. {MIN=0, MAX=9999}");
-					cvpv[i].strvalue = Z_StrDup(lua_tostring(L, 5));
+					char *strvalue = Z_StrDup(lua_tostring(L, 5));
+					if (fasticmp(strvalue, "MIN"))
+						hasminmax = true;
+					cvpv[i].strvalue = strvalue;
 					cvpv[i].value = (INT32)lua_tonumber(L, 6);
 					i++;
 					lua_pop(L, 1);
 				}
 				cvpv[i].value = 0;
 				cvpv[i].strvalue = NULL;
+				hasminmax = false;
```

Comment:
Is this line intended? It will cause `hasminmax` always be false

Reply:
uhhhhhhhhhhhhhh somehow it works but ye shouldnt be there lul

Reply:
oh wait it didnt work lmfao oops

Reply:
hm tbh with it fixed, i dont really like the sliders

Reply:
so uhhhhhh, personally id just let it not be a slider

Reply:
![blobcatpat](https://github.com/Indev450/SRB2Kart-Saturn/assets/44119190/d403eb41-a727-4462-bfc4-fe64b2969951)